gitkit/github.go

```diff
@@ -36,19 +36,14 @@ func NewGitHubClient() *GitHubClient {
 	}
 }
 
-type CloneResult struct {
+type GitHubResult struct {
 	RepoName  string
 	RepoURL   string
 	LocalPath string
-	Status    string
+	Type      string
```

Comment:
Not clear what `Type` means on it's own `DownloadCommand` or something says more.

Reply:
Changed to `GitCommand`